pom.xml

```diff
@@ -31,7 +31,7 @@
 
     <properties>
         <inceptionYear>2022</inceptionYear>
-        <opentelemetry.spec.version>1.13.0</opentelemetry.spec.version>
+        <opentelemetry.spec.version>1.14.0</opentelemetry.spec.version>
```

Comment:
spec version 1.14 will come only with otel java version 1.19

Reply:
> spec version 1.14 will come only with otel java version 1.19

Yes, and that is close, right?  opentelemety-java 1.19.0 is out already. But opentelemetry-java-instrumentation version 1.19.0 is still missing only. If we can not wait for that, a step in this PR back to spec 1.13.0 would be theoretical possible, but then the config property must be set to the experimental one (otel.experimental.sdk.enabled=true) - that than should be declared deprecated and the new one should be mentioned in the statement commenting the configuration. TCK tests for this might be usefull too.

Reply:
the opentelemetry-instrumentation-bom already depends on the opentelemetry-bom

Reply:
By the way, I just double checked and confirmed that the current main branch is up to date with 1.13 and I think we also use opentelemetry 1.18.0 already. We need to hold this one until everything we need from OTel is in place. As mentioned on today's call, I suggest once we have all TCKs are passing, we can then move up to a new level.

Reply:
When we move up to 1.14, we also need to update the spec doc to mention the new configuration property.

Reply:
I will do an update on this PR to change the parameter in he TCK soon.

Reply:
I did the update on a separate 2nd PR https://github.com/eclipse/microprofile-telemetry/pull/64, but with the changes meanwhile, I have to redo them.

I will start rebasing this one...

But then, [as metioned here](https://github.com/eclipse/microprofile-telemetry/pull/64#issuecomment-1278635514) a decision need to done, what sould be part of this PR and what on separate ones.